darwin/importer/importer.py

```diff
@@ -121,7 +127,7 @@ def maybe_console(*args: Union[str, int, float]) -> None:
 
     maybe_console(f"Found {len(files)} files")
 
-    if use_multi_cpu:
+    if use_multi_cpu and MPIRE_AVAILABLE and cpu_limit > 1:
```

Comment:
Changes this to only use mcpu if:

* The settings have been set
* The module is actually available
* There is over one core (This in particular stops issues with AWS Lambda setups)